input/resources/au-erequesting-server.xml

```diff
@@ -38,7 +38,7 @@
   </implementationGuide>
   <rest>
     <mode value="server"/>
-    <documentation value="**SHALL** support ServiceRequest."/>
+    <documentation value="The AU eRequesting Server **SHALL**:&#xA;1. Support the ServiceRequest resource profiles.&#xA;1. Implement the RESTful behavior according to the FHIR specification.&#xA;1. Declare a CapabilityStatement identifying the list of profiles, operations, and search parameters supported."/>
```

Comment:
1. Suggest remove "Declare a CapabilityStatement identifying ...." 
1. Adjust the first point on ServiceRequest to match the requirements documented for ServiceRequest. Rather than "SHALL support the ServiceRequest resource profiles" it is "Support at least one AU eRequesting ServiceRequest profile, and the conformance expectations for the ServiceRequest resource."

Reply:
Done